src/banded/BandedMatrix.jl

```diff
@@ -1007,3 +1007,57 @@ function resize(A::BandedSubBandedMatrix, n::Integer, m::Integer)
     l,u = bandwidths(A)
     _BandedMatrix(reshape(resize!(vec(copy(bandeddata(A))), (l+u+1)*m), l+u+1, m), n, l,u)
 end
+
+function Base.sum(A::BandedMatrix)
+    l,u = bandwidths(A)
+    #For empty matrices
+    if(l + u < 0)
+        return convert(eltype(A),0)
+    end
+    height,width = size(A)
```

Comment:
This is too verbose... calling this `m,n` would be cleaner and make the code easier to read.